skimage/exposure/hdr.py

```diff
@@ -0,0 +1,326 @@
+import numpy as np
+
+
+def get_radiance(ims, exp, radiance_map):
```

Comment:
I suggest here and elsewhere the following variables

def get_radiance(images, exposures, radiance_map):